cpp/src/io/parquet/reader_impl_helpers.hpp

```diff
@@ -121,9 +124,13 @@ struct metadata : public FileMetaData {
   void sanitize_schema();
 };
 
+using arrow_schema_t = std::unordered_map<std::string, cudf::data_type>;
```

Comment:
I've been told in the past to prefer `_type` to `_t`, but not sure if that applies here.

Reply:
Thanks for pointing this out. I am still not sure if to keep or build this instead of co-walking schema when building output columns

Reply:
I went ahead with co-walking the schemas and adding an `arrow_type` field in `SchemaElement` as you suggested so this is removed from the code now!